debian/portal.triggers

```diff
@@ -1,7 +1,7 @@
 # Register interest in Python interpreter changes (Python 2 for now); and
 # don't make the Python package dependent on the virtualenv package
 # processing (noawait)
-interest-noawait /usr/bin/python2.7
+interest-noawait /usr/bin/python3.5
```

Comment:
isn't `/usr/bin/python3` a symbolic link to the desired version 3 installed?

Reply:
I'm not 100% sure why this references a specific version (I copy/pasted this change from [their python3 docs](https://dh-virtualenv.readthedocs.io/en/1.1/howtos.html#building-packages-for-python3)) but it seems like [`deb-triggers`](http://man7.org/linux/man-pages/man5/deb-triggers.5.html) takes `/usr/bin/python3.5` as the `trigger-name`. 

Perhaps there's a pre-defined list `trigger-name` is from, and `trigger-name` is not a binary/symlink?